core/node/eth_watch/src/client.rs

```diff
@@ -65,6 +65,8 @@ pub trait EthClient: 'static + fmt::Debug + Send + Sync {
         hashes: Vec<H256>,
     ) -> EnrichedClientResult<Vec<Option<Vec<u8>>>>;
 
+    async fn get_settlement_layer(&self) -> Result<Address, ContractCallError>;
```

Comment:
Dumb question: It doesn't look like this method is used; the only use is in proxying `ZkSyncExtentionEthClientW` implementation.